### ISSUE TITLE
Fix geo ordering

### ISSUE THREAD

**Issue by temi4** [barseghyanartur/django-elasticsearch-dsl-drf#64]
Hello, please add fix:

I use GeoSpatialOrderingFilterBackend. In example write:
```
geo_spatial_ordering_fields = {
        'location': {
            'field': 'point',
       }    
}
```
in function 'get_geo_distance_params' param 'field' get full data `{'field': 'point',}`, don't 'field name'
or may be correct example:
```
geo_spatial_ordering_fields = {
        'location': 'point',
}
```
And __ordering_params.append(__params) don't add '_geo_distance' value
https://www.elastic.co/guide/en/elasticsearch/guide/current/sorting-by-distance.html
```
__params = {
          '_geo_distance': __params
}
```
http://take.ms/6tFnM

**Comment by barseghyanartur** [comment]
@temi4:

Thanks for reporting this. I'll check it ASAP.

**Comment by barseghyanartur** [comment]
@temi4:

It seems that there has been a change in 5.x due to which it didn't work correctly anymore.

It's fixed now.

Additionally, although it wasn't shown like that in the docs, for consistency with filters I changed the accepted formats.

You can now write as follows:

Example 1:

```python
 geo_spatial_ordering_fields = {
     'location': None,
 }
```

Example 2:

```python
geo_spatial_ordering_fields = {
    'location': 'location',
}
```

Example 3:

```python
geo_spatial_ordering_fields = {
    'location': {
        'field': 'location'
    },
}
```

This change will be present in 0.12 (next release).

**Comment by temi4** [comment]
@barseghyanartur Thank, for fast fix.

**Comment by barseghyanartur** [comment]
Solved in 0.12.
